src/contracts/proxies/ODSafeManager.sol

```diff
@@ -160,11 +164,17 @@ contract ODSafeManager is IODSafeManager {
   function modifySAFECollateralization(
     uint256 _safe,
     int256 _deltaCollateral,
-    int256 _deltaDebt
+    int256 _deltaDebt,
+    bool _useSafeHandlerAddress
   ) external safeAllowed(_safe) {
     SAFEData memory _sData = _safeData[_safe];
+    if (_deltaDebt != 0) {
+      ITaxCollector(taxCollector).taxSingle(_sData.collateralType);
+    }
+    address collateralSource = _useSafeHandlerAddress ? _sData.safeHandler : msg.sender;
```

Comment:
can you reverse this check:
`bool _nonSafeHandlerAddress` so the default, using the safeHandler, will be zero. it will be slightly more gas efficient to not pass non-zero